### DISCUSSION TITLE
pax4pro.com

### DISCUSSION THREAD

**Discussion by Gr00nd3** [community/community#116347] [Enterprise] [answered]
### Select Topic Area

Show & Tell

### Body

I own this domain and website, which is hacked I believe. 
my question is to understand the link before the actual link to my site:

https://l.wl.co/l?u=https://pax4pro.com

when I open the site anyhow it requires the username and password which I do know them both and has not changed since the last I recal. 

 I would appreciate any feedback on the link comes before the site link. 
 
![IMG_0222](https://github.com/community/community/assets/165542432/3095390f-788f-4dcb-ba0c-269e50b934bf)

**Comment by Akash1134** [comment, marked as answer]
Hi @Gr00nd3 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as this one. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
